guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -52,14 +52,16 @@ double in size before starting the migration process.
 Check the size of /var/lib/pulp/published with 'du -sh /var/lib/pulp/published/'
 [OK]
 ----
-ifdef::satellite[]
+
 . Update the file permissions before upgrading {ProjectServer} using the following command:
 +
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
 # {foreman-maintain} content prepare
 ----
-endif::[]
++
+. As part of the final step of `{foreman-maintain} content prepare`, {Project} checks whether any content units are unmigrated.
+If `{foreman-maintain} content migration-stats` identifies corrupted or missing content, you might see something similar to the following:
```

Comment:
@jlsherrill So maybe we should rather say on this line:
`If {Project} identifies corrupted or missing content in the migration statistics, you might see...`
Because if we mention the command as above, it looks like the user has to run it.
Am I understanding this correctly?